src/Storefront/Page/Robots/Parser/RobotsDirectiveParser.php

```diff
@@ -143,4 +144,26 @@ public function parse(string $text, Context $context, ?string $salesChannelId =
 
         return $parsingEvent->parsedResult;
     }
+
+    /**
+     * @deprecated tag:v6.8.0.0 - reason:becomes-obsolete - will be removed, as string parsing is deprecated
+     */
+    public static function parseDirectiveFromString(string $line): ?RobotsDirective
+    {
+        Feature::triggerDeprecationOrThrow(
+            'v6.8.0.0',
+            Feature::deprecatedMethodMessage(self::class, __METHOD__, 'v6.8.0.0', 'will be removed, as string parsing is deprecated')
```

Comment:
this will create a message like this:

`Use "will be removed, as string parsing is deprecated" instead.`